.github/workflows/serverless-project.yml

```
@@ -75,7 +75,7 @@ jobs:
           project-id: "elastic-observability-ci"
 
       - name: Set up Cloud SDK
-        uses: google-github-actions/setup-gcloud@77e7a554d41e2ee56fc945c52dfd3f33d12def9a
+        uses: google-github-actions/setup-gcloud@26f734c2779b00b7dda794207734c511110a4368
```

In order to improve traceability
```suggestion
        uses: google-github-actions/setup-gcloud@26f734c2779b00b7dda794207734c511110a4368 # v3.0.0
```